test/unit/puppetlabs/services/config/puppet_server_config_service_test.clj

```diff
@@ -19,9 +19,11 @@
    profiler/puppet-profiler-service])
 
 (def required-config
-  (merge (jruby-testutils/jruby-puppet-tk-config
-           (jruby-testutils/jruby-puppet-config {:max-active-instances 1}))
-         {:webserver    {:port 8081}}))
+  (-> (jruby-testutils/jruby-puppet-tk-config
+        (jruby-testutils/jruby-puppet-config {:max-active-instances 1}))
+      (assoc-in [:webserver :port] 8081)
+      (assoc-in [:jruby-puppet :master-conf-dir]
+                (str test-resources-dir "/master/conf"))))
```

Comment:
I'm getting 

```
Exception in thread "main" java.lang.RuntimeException: Unable to resolve symbol: test-resources-dir in this context, compiling:(puppetlabs/services/config/puppet_server_config_service_test.clj:26:17)
 ```
because `test-resources-dir` is defined below. I think that needs to be moved up.